benchmark/opperf/utils/op_registry_utils.py

```diff
@@ -240,6 +242,26 @@ def get_all_reduction_operators():
     return reduction_mx_operators
 
 
+def get_all_sorting_searching_operators():
+    """Gets all Sorting and Searching operators registered with MXNet.
+
+    Returns
+    -------
+    {"operator_name": {"has_backward", "nd_op_handle", "params"}}
+    """
+    sort_search_ops = ['sort', 'argsort', 'argmax', 'argmin', 'topk']
```

Comment:
If `argmax` and `argmin` are treated as sort search ops, why not `max` and `min`?

Reply:
`max` and `min` have already been covered in reduction op
In that sense, `argmax` and `argmin` can be placed in 
https://github.com/apache/incubator-mxnet/blob/b88705e7744be1cc22122eb2c5b492e495ab45bb/benchmark/opperf/nd_operations/reduction_operators.py#L24

Reply:
NDArray API shows - argmax and argmin in the "Array sorting and searching" section.
So I guess this distinction is correct
Sort & Search - `argmax`, `argmin`
Reduction - `max`, `min`
![Screen Shot 2019-07-15 at 4 13 36 PM](https://user-images.githubusercontent.com/10992635/61255011-ad99bc80-a71b-11e9-8ee9-836d3f1cabf0.png)
https://mxnet.incubator.apache.org/api/python/ndarray/ndarray.html#array-sorting-and-searching

Reply:
Lin, reasoning was same as described above the Chai.